js/app.js

```diff
@@ -198,45 +181,44 @@
       setTime = Date.now();
       console.log(setTime);
     }, time);
-
   }
 
+  //    set interval function to change the number of seconds every ONE seconds
   function startCount () {
     countDown = setInterval(function () {
       secondPass();
     }, 1000);
   }
 
-
-  var setTimeoutTwo;
-
+  //   start counting When the box is clicked
   function clickMeEngine () {
 
+    numberOfClicks++;
     endTime = Date.now();
-    reactTime = (endTime - setTime) / 1000;
+    reactionTime = (endTime - setTime) / 1000;
 
-    if (reactTime >= 60) {
+    if (reactionTime >= 60) {
 
-      var myTimeMin = Math.floor(reactTime / 60),
-        remMyTimeSec = Math.floor(reactTime % 60);
+      var myTimeMin = Math.floor(reactionTime / 60),
+        remMyTimeSec = Math.floor(reactionTime % 60);
 
-        elements.playerTime.innerHTML = myTimeMin + 'm, ' + remMyTimeSec + 's';
+      elements.playerTime.innerHTML = myTimeMin + 'm, ' + remMyTimeSec + 's';
 
-    } else if (reactTime >= 1) {
+    } else if (reactionTime >= 1) {
 
-        elements.playerTime.innerHTML = reactTime + 's';
+      elements.playerTime.innerHTML = reactionTime + 's';
 
-        var comments = [
-            "Congrats," + "\n" + "You are faster than a TURTLE!",
-            "Come On," + "\n" +"Is That All You Have?!",
-            "Do you even know who FLASH is?!",
-            "ooh" + "\n" + "I slept twice"
-        ];
-        alert(comments[Math.round(Math.random() * 3)]);
+      var comments = [
+        'Congrats,\n You are faster than a TURTLE!',
```

Comment:
Following your instructions, well, and you 'do NOT betting on things you don't know' :smile: